auslib/blobs/schemas/systemaddons.yml

```diff
@@ -0,0 +1,71 @@
+---
+  title: GMP Blob
+  type: object
+  required:
+    - name
+    - schema_version
+    - hashFunction
+    - vendors
```

Comment:
Now that this is just a SystemAddons blob, we can rename this to something better - perhaps "addons"? Please update the references in the descriptions below as well.